src/Post_List_Field.php

```diff
@@ -285,7 +285,7 @@ private function get_manually_selected_posts( $value ): array {
 			$item = [];
 
 			// No post and no override/custom
-			if ( empty( $row[ self::FIELD_MANUAL_POST ] ) || empty( $row[ self::FIELD_MANUAL_TOGGLE ] ) ) {
+			if ( empty( $row[ self::FIELD_MANUAL_POST ] ) && empty( $row[ self::FIELD_MANUAL_TOGGLE ] ) ) {
```

Comment:
hrmmm this was previously changed [due to a critical error ](https://github.com/moderntribe/tribe-acf-post-list-field/commit/fa611911e074f836b6508b01028abfa7d5d5ed21#diff-b6deb4a346bfca56c213531c23971c64b784843b286d6a5d0ed9dcd270fab809R288).

The problem here is `$row[ self::FIELD_MANUAL_POST ]` can never been empty as [it's used right after this check](https://github.com/moderntribe/tribe-acf-post-list-field/blob/main/src/Post_List_Field.php#L293). 

So perhaps we move the empty checks to their respective if statements below?